composer.json

```diff
@@ -39,6 +39,7 @@
         "willvincent/laravel-rateable": "^2.0"
     },
     "require-dev": {
+        "imanghafoori/laravel-microscope": "^1.0",
```

Comment:
I'd be happy to merge this PR, but we don't need to add this package permanently. Thanks!